src/styles/index.ts

```diff
@@ -3740,8 +3740,9 @@ const styles = (theme: ThemeColors) =>
 
         searchInputStyle: {
```

Comment:
Why are we using something called "searchInputStyle" here? And then why are we changing all of the existing classes styles? 

We should have a separate class for something like `tableHeaderText` or something, and we shouldn't be touching this.

Reply:
@shubham1206agra can you show me what any other place that used this style looks like now? I am very surprised we went this route instead of writing a new style... am I missing something?